testing/cucumber/features/linters/test_name_too_long.feature

```diff
@@ -0,0 +1,41 @@
+Feature: Scenario name with too many characters
+
+  As a reader of documentation
+  I want test names not to be unduly long
+  So that I can easily understand its purpose
+
+
+  Scenario: Linter
+
+    Given a linter for test names with too many characters
+    And the following feature:
+      """
+      Feature:
+
+        Scenario: tea exists and teapots exist and so do cups and saucers and there might be milk in the milk jug together with sugar cubes
+      """
+    When it is linted
+    Then an error is reported:
+      | linter                              | problem                                                  | location         |
+      | TestNameWithTooManyCharactersLinter | Scenario name is too long. 121 characters found (max 80) | <path_to_file>:4 |
+
+
+  Scenario: Configuration of test name count threshold
+
+    Given a linter for test names with too many characters has been registered
+    And the following configuration file:
+      """
+      TestNameWithTooManyCharactersLinter:
+        TestNameLengthThreshold: 80
```

Comment:
Because this scenario is demonstrating configuring the linter, it would be better to configure the limit to a value that is not already the default value so that there is a visible difference in the output. Go with `30` or something.

Reply:
changed to 30